scripts/install

```diff
@@ -49,6 +49,7 @@ ynh_app_setting_set --app=$app --key=domain --value=$domain
 ynh_app_setting_set --app=$app --key=path --value=$path_url
 ynh_app_setting_set --app=$app --key=admin --value=$admin
 ynh_app_setting_set --app=$app --key=password --value=$password
+ynh_app_setting_set --app=$app --key=architecture --value=$architecture
```

Comment:
Zblerg same comment as in another app, it's weird to save this info as a setting :/